README.md

```diff
@@ -7,7 +7,7 @@ MarkLogic Data Hub is a data integration platform and toolset that helps you qui
 
 # Version Support
 
-For Data Hub version 5.0, you need:
+For Data Hub version 5.2, you need:
```

Comment:
This information shouldn't be listed here - link to https://docs.marklogic.com/datahub/refs/version-compatibility.html instead.